### DISCUSSION TITLE
更好的保存双语翻译后的网页

### DISCUSSION THREAD

**Discussion by Zj-lu** [immersive-translate/immersive-translate#2283] [Ideas]
### 新功能 | New Feature

一直习惯使用网页看论文 但是有点不好的是使用双语翻译之后不管是保存为ttml还是pdf都是会出现和原网站不一样的问题  要是可以整个将网站内的双语翻译全部拿走那种就好了  可以随时随地离线 节省一次次打开在线网页 重新翻译的字符 

### 为什么您想要添加这些功能 | Why you want to add these features

亦或者双语翻译pdf输出格式和在线时候是一样的也是一段一段对应而不是整页对应  这样可能更加的贴近双语对照设计的初衷吧

### 提议 | Proposal

优化双语对照后的排版即可

**Comment by keyboard3** [comment]
![QQ_1731679169957](https://github.com/user-attachments/assets/f265f028-fb49-45f6-beef-2b10e2460dd9)
可以使用网页快照功能呀
